news/2024/2024-05-20-the-followpoint-yesmydarknesss-the-osu-catch-prodigy.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "The Followpoint: YesMyDarknesss, the osu!catch Prodigy"
+date: 2024-05-20 11:00:00 +0000
+---
+
+Read up on the latest interview with YesMyDarknesss, the osu!catch #1 player.
+
+![](/wiki/shared/news/2024-05-20-the-followpoint-yesmydarknesss-the-osu-catch-prodigy/banner.jpg)
+
+Welcome or welcome back to The Followpoint, a "community spotlight" series where we interview various players and community figures to get to know them better!
+
+We are back with a second interview of our special set, interviewing the very top of each game mode, this time featuring YesMyDarknesss (a.k.a Hyperw7), osu!catch rank #1 player!
```

Comment:
```suggestion
We are back with a second interview of our special set, interviewing the very top of each game mode, this time featuring YesMyDarknesss (a.k.a Hyperw7), osu!catch #1 player!
```

Reply:
Thing is I used `rank #1 player` because that's what Wala made it in a previous newspost

Reply:
well yeah i'd subjectively prefer it that way

Reply:
I don't mind either, so it's up to news team really